i18n/zh/docusaurus-plugin-content-docs/current/design-document/03-connect/11-kafka-connector.md

```diff
@@ -20,7 +20,7 @@ pubSubConfig:
   passWord: kafkaPassWord
 connectorConfig:
   connectorName: kafkaSink
-  # kafka连接参数 ↓
+  # kafka 连接参数 ↓
   bootstrapServers: 127.0.0.1:9092
   topic: TopicTest
```

Comment:
Sometimes, we don't want to format the content within a code block.

Reply:
I need to update AutoCorrect to support this as you like.

We can just revert this file changes.

Reply:
No rush. There are no manual modifications in this PR. You can simply rerun the formatting in this PR after improving your tool. ☺️

Reply:
You may notice that the highlighted comment in this conversation uses manual line breaks (displayed as `↓` in Word and represented by `^l`. The standard line break should be `^p`.) These manual line breaks might have been introduced by contributors copying content from QQ or webpages (since messages sent from Windows QQ use `\r` instead of `\n`). These line breaks should not appear in markdown documents, so it would be great if you could support replacing them with standard line breaks.

Reply:
I have update for this.